common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenResponse.java

```diff
@@ -62,10 +62,14 @@ public BrokerTokenResponse(MicrosoftStsTokenResponse microsoftStsTokenResponse){
         setClientId(microsoftStsTokenResponse.getClientId());
         setExtExpiresIn(microsoftStsTokenResponse.getExtExpiresIn());
         setFamilyId(microsoftStsTokenResponse.getFamilyId());
+        setSpeRing(microsoftStsTokenResponse.getSpeRing());
+        setRefreshTokenAge(microsoftStsTokenResponse.getRefreshTokenAge());
+        setServerErrorCode(microsoftStsTokenResponse.getServerErrorCode());
```

Comment:
Q: 
Should these
`setServerErrorCode(microsoftStsTokenResponse.getServerErrorCode());
setServerSubErrorCode(microsoftStsTokenResponse.getServerSubErrorCode());`
move to MicrosoftTokenErrorResponse?

The token result contains two properties
1. TokenResponse for the successful response
 2. TokenErrorResponse for the error response

Reply:
Synced offline. This code just ensures that data isn't "lost" when parceling